lib/Doctrine/Common/Persistence/Mapping/Driver/SymfonyFileLocator.php

```diff
@@ -54,15 +54,24 @@ class SymfonyFileLocator implements FileLocator
     protected $fileExtension;
 
     /**
+     * Represents PHP namespace delimiters when looking for files
+     *
+     * @var string
+     */
+    protected $nsSeparator;
```

Comment:
please make it private.
Anything protected is an extension point for which we must maintain BC, so it is better to use private visibility when introducing new properties/methods.
Increasing the visibility is easy (but may not be desirable which is why we are always asking for justification in this case to evaluate whether it is the right way), but the opposite (turning a protected member into a private one) is a BC break (and so is forbidden except in major versions)